app/user.go

```diff
@@ -0,0 +1,597 @@
+}
+
+func (c *UserClient) setNamespacePermissions(
+	ctx *cli.Context,
+	userID string,
+	userEmail string,
+	namespacePermissions []string,
+) error {
+	npm, err := toNamespacePermissionsMap(namespacePermissions)
+	if err != nil {
+		return err
+	}
+	for namespace, permission := range npm {
+		ag, err := toNamespaceActionGroup(permission)
+		if err != nil {
+			return err
+		}
+		if err := c.updateUserNamespacePermissions(ctx, userID, userEmail, namespace, ag); err != nil {
+			return err
+		}
+	}
+	return nil
+}
+
+func (c *UserClient) deleteNamespacePermission(
+	ctx *cli.Context,
+	userID string,
+	userEmail string,
+	namespaces []string,
+) error {
+	for _, ns := range namespaces {
+		if err := c.updateUserNamespacePermissions(ctx, userID, userEmail, ns, auth.NAMESPACE_ACTION_GROUP_UNSPECIFIED); err != nil {
+			return err
+		}
+	}
+	return nil
+}
+
+func toNamespacePermissionsMap(keyValues []string) (map[string]string, error) {
+	res := map[string]string{}
+	for _, kv := range keyValues {
+		parts := strings.Split(kv, "=")
+		if len(parts) != 2 {
+			return nil, fmt.Errorf("invalid namespace permission \"%s\" must be of format: \"namespace=permission\"", kv)
+		}
+
+		namespace := parts[0]
+		actionGroupValue := parts[1]
+
+		if len(namespace) == 0 {
+			return nil, errors.New("namespace must not be empty in namespace permission")
+		}
+
+		res[namespace] = actionGroupValue
+	}
+	return res, nil
+}
+
+func NewUserCommand(getUserClientFn GetUserClientFn) (CommandOut, error) {
+	var c *UserClient
+	return CommandOut{
+		Command: &cli.Command{
+			Name:    "user",
+			Aliases: []string{"u"},
+			Usage:   "User management operations",
+			Before: func(ctx *cli.Context) error {
+				var err error
+				c, err = getUserClientFn(ctx)
+				return err
+			},
+			Subcommands: []*cli.Command{
+				{
+					Name:    "list",
+					Usage:   "List users",
+					Aliases: []string{"l"},
+					Flags: []cli.Flag{
+						&cli.StringFlag{
+							Name:    NamespaceFlagName,
+							Usage:   "List users that have permissions to the namespace",
+							Aliases: []string{"n"},
+						},
+					},
+					Action: func(ctx *cli.Context) error {
+						return c.listUsers(ctx.String(NamespaceFlagName))
+					},
+				},
+				{
+					Name:    "get",
+					Usage:   "Get user information",
+					Aliases: []string{"g"},
+					Flags: []cli.Flag{
+						userIDFlag,
+						userEmailFlag,
+					},
+					Action: func(ctx *cli.Context) error {
+						n, err := c.getUser(ctx.String(userIDFlagName), ctx.String(userEmailFlagName))
+						if err != nil {
+							return err
+						}
+						return PrintProto(n)
+					},
+				},
+				{
+					Name:    "invite",
+					Usage:   "Invite users to Temporal Cloud",
+					Aliases: []string{"i"},
+					Flags: []cli.Flag{
+						&cli.StringSliceFlag{
+							Name:     userEmailFlagName,
+							Usage:    "The email address of the user, you can supply this flag multiple times to invite multiple users in a single request",
```

Comment:
nit: `StringSliceFlag` mentions that it accepts multiple inputs so I think we could shorten this to `The email address(es) of the user to invite`